walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,141 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void everyAttendeeIsConsideredOptionalNot() {
+    // Have each person have different events. We should see two options because each person has
+    // split the restricted times.
+    //
+    // Events  : |-----|--A--|--C--|--B--|-----|
```

Comment:
It's hard to visualize that `C` has an all-day event here.  Maybe move `C` to their own line?